proto/xion/dkim/v1/state.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package xion.dkim.v1;
+
+import "cosmos/orm/v1/orm.proto";
+
+option go_package = "github.com/burnt-labs/xion/x/dkim/types";
+
+// https://github.com/cosmos/cosmos-sdk/blob/main/orm/README.md
+
+message DkimPubKeys {
+    option (cosmos.orm.v1.table) = {
+      id: 1;
+      primary_key: { fields: "domain" }
+      index: { id: 1 fields: "pub_key" unique: true }
+    };
+
+    string domain = 1;
+    string pub_key  = 2;
```

Comment:
Part of the DKIM record is the "selector" or `s` record. We are not storing that, nor using it in the component key.